packages/commons/src/AxeResult.cs

```diff
@@ -3,25 +3,71 @@
 
 namespace Deque.AxeCore.Commons
 {
+    /// <summary>
+    /// Axe results object
+    /// </summary>
     public class AxeResult
     {
+        /// <summary>
+        /// These results indicate what elements failed the rules.
+        /// </summary>
         public AxeResultItem[] Violations { get; }
+
+        /// <summary>
+        /// These results indicate what elements passed the rules.
+        /// </summary>
         public AxeResultItem[] Passes { get; }
+
+        /// <summary>
+        /// These results indicate which rules did not run because no matching content was found on the page. 
+        /// For example, with no video, those rules won't run.
+        /// </summary>
         public AxeResultItem[] Inapplicable { get; }
+
+        /// <summary>
+        /// These results were aborted and require further testing. 
+        /// This can happen either because of technical restrictions to what the rule can test, or because a javascript error occurred.
+        /// </summary>
         public AxeResultItem[] Incomplete { get; }
+
+        /// <summary>
+        /// The date and time that analysis was completed.
+        /// </summary>
         public DateTimeOffset? Timestamp { get; private set; }
+
+        /// <summary>
+        /// Information about the current browser or node application that ran the audit.
+        /// </summary>
         public AxeTestEnvironment TestEnvironment { get; private set; }
+
+        /// <summary>
+        /// The runner that ran the audit.
+        /// </summary>
         public AxeTestRunner TestRunner { get; set; }
 
+        /// <summary>
+        /// The URL of the page that was tested.
+        /// </summary>
         public string Url { get; private set; }
 
+        /// <summary>
+        /// The Error that was found on the application that ran the audit.
+        /// </summary>
         public string Error { get; private set; }
 
-
+        /// <summary>
+        /// The Name of the application that ran the audit.
+        /// </summary>
         public string TestEngineName { get; private set; }
 
+        /// <summary>
+        /// The Version of the application that ran the audit.
+        /// </summary>
         public string TestEngineVersion { get; private set; }
 
+        /// <summary>
+        /// The tool options used for the configuration of the data format used by axe.
```

Comment:
It seemed like the ToolOptions mapped most closely to the axe configurations in https://www.deque.com/axe/core-documentation/api-documentation/#api-name-axeconfigure, so I used that description